### PR TITLE
[Android/Build] clean build script

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. add flag to disable tf-lite.
2. update build script for each build option.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: api/android/build-android-lib.sh [hunk 1]
```diff
@@ -13,39 +13,49 @@
...
 # For example, to build library with core plugins for arm64-v8a
 # ./build-android-lib.sh --api_option=lite --target_abi=arm64
 #
 
-# API build option ('lite' to build with GStreamer core plugins)
+# API build option
+# 'all' : default
+# 'lite' : with GStreamer core plugins
+# 'single' : no plugins, single-shot only
+# 'internal' : no plugins, single-shot only, disable SNAP and tf-lite
```
Comment (dongju-chae) [inline api/android/build-android-lib.sh]: How about this? `all`, `core`, `single`, `single-internal`

Comment (jaeyun-jung) [reply api/android/build-android-lib.sh]: Build for internal release : I am not sure internal release includes only single-shot functions. (TBD)
About the 'lite', in the past we determined the lib name as nnstreamer-lite with minimized plugins.

**Comment by jaeyun-jung** [comment]
2020-03-17 PR updated
- cleanup tf-lite module in mk

**Comment by jaeyun-jung** [comment]
2020-03-18 PR updated
- set default abi arm64-v8a
- update document

File: api/android/api/src/main/jni/Android.mk [hunk 2]
```diff
@@ -82,4 +93,9 @@ GSTREAMER_INCLUDE_CA_CERTIFICATES := no
 
 include $(GSTREAMER_NDK_BUILD_PATH)/gstreamer-1.0.mk
 
+#------------------------------------------------------
+# NDK cpu-features
+#------------------------------------------------------
+ifeq ($(ENABLE_TF_LITE), true)
```
Comment (wooksong) [inline api/android/api/src/main/jni/Android.mk]: Could we eliminate this ifeq (99) and above ifeq (69)? For example, 

```
LOCAL_EXTRA_CFLAGS :=
ifeq ($(ENABLE_TF_LITE), true)
include $(LOCAL_PATH)/Android-tensorflow-lite.mk
LOCAL_EXTRA_CFLAGS += -DENABLE_TF_LITE=1
endif
...omission...
LOCAL_CFLAGS += LOCAL_EXTRA_CFLAGS 
```
How about that?

Comment (jaeyun-jung) [reply api/android/api/src/main/jni/Android.mk]: added new flags as you commented (NNS_API_FLAGS and NNS_API_STATIC_LIBS)

**Pull request merged**